src/tools/prefs.win.cpp

```diff
@@ -99,10 +101,18 @@ static bool save_prefs(fs::path fpath) {
 static bool load_prefs(fs::path fpath) {
 	prefsDirty = false;
 	std::map<std::string,boost::any> temp_prefs;
-	std::ifstream fin(fpath.string().c_str());
+	std::istream* in;
```

Comment:
This should be an `std::unique_ptr<std::istream>`. (If that causes problems for some reason, `shared_ptr` is also acceptable.)

Reply:
Ah, looks like you already made that change in a later commit, so you can disregard this.